pkg/server/backend_manager.go

```diff
@@ -85,7 +87,25 @@ type backend struct {
 func (b *backend) Send(p *client.Packet) error {
 	b.mu.Lock()
 	defer b.mu.Unlock()
-	return b.conn.Send(p)
+	errChan := make(chan error, 1)
+	go func() {
```

Comment:
We should add a comment for why we have an explicit timeout with a link to https://github.com/grpc/grpc-go/issues/1229

Reply:
Done. Thanks!